staty/exceptions.py

```diff
@@ -41,12 +41,12 @@ class MissingHandlerException(StatyBaseException):
 
 
 class HTTPError(StatyBaseException):
-    def __init__(self, *args, **kwargs):
-        self.response = kwargs.pop("response", None)
-        self.request = kwargs.pop("request", None)
+    def __init__(self, response=None, request=None, *args):
```

Comment:
it would be great if there was an explicit `message` argument

the`actual use I have for this exception is:

```python
raise HTTPError('cute error message', response=response)
```